sql_generators/mobile_kpi_support_metrics/templates/attribution_clients.bigconfig.yml

```diff
@@ -1,22 +1,59 @@
 type: BIGCONFIG_FILE
-tag_deployments:
-- collection:
-    name: {{ bigeye_collection }}
-  deployments:
-  - column_selectors:
-    - name: {{ project_id }}.{{ project_id }}.{{ dataset }}_derived.{{ name }}_{{ version }}.*
-    metrics:
-    - metric_type:
+
+tag_definitions:
+  - tag_id: UNIQUE_FIELDS
+    column_selectors:
+      - name: {{ project_id }}.{{ project_id }}.{{ dataset }}_derived.{{ name }}_{{ version }}.client_id
+  - tag_id: ALL_FIELDS
+    column_selectors:
+      - name: {{ project_id }}.{{ project_id }}.{{ dataset }}_derived.{{ name }}_{{ version }}.*
+
+saved_metric_definitions:
+  metrics:
+    - saved_metric_id: is_not_null
+      metric_type:
+        predefined_metric: PERCENT_NULL
+      threshold:
+        type: CONSTANT
+        upper_bound: 0
+      metric_schedule:
+        named_schedule:
+          name: default
+    - saved_metric_id: is_unique
+      metric_type:
+        predefined_metric: COUNT_DUPLICATES
+      threshold:
+        type: CONSTANT
+        upper_bound: 0
+      metric_schedule:
+        named_schedule:
+          name: default
```

Comment:
Im not sure on if default is 13:00 UTC

Reply:
yes it is, I created it in the UI to make it less verbose in the configuration since we have to repeat it quite often.